extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -426,6 +438,14 @@ export class AmpPanZoom extends AMP.BaseElement {
       transform: translate(this.posX_, this.posY_) +
           ' ' + scale(this.scale_),
     });
+    const transformEndEvent =
+    createCustomEvent(this.win, `${TAG}.transformEnd`, {
+      scale: this.scale_,
+      x: this.posX_,
+      y: this.posY_,
+    });
+    this.action_.trigger(this.element, 'transformEnd', transformEndEvent,
+        ActionTrust.HIGH);
```

Comment:
Why should this be `HIGH` trust while the other event is `LOW`?

Reply:
A common use case of the transformEnd event is to trigger amp-bind / set state. That requires `ActionTrust.HIGH`.

Reply:
Correct me if I'm wrong, but `transform` requires `LOW` and will subsequently trigger `transformEnd` with `HIGH` trust.

Isn't this essentially a mechanism to "upgrade" event trust? If so then that's a little concerning.

Reply:
Yeah you're right. We should set `transform` to min-trust = `HIGH`.